jenkins/ci.suse.de/templates/cloud-heat-template.yaml

```diff
@@ -100,10 +115,10 @@
     pipeline-scm:
       scm:
         - git:
-            url: ${git_automation_repo}
+            url: ${{git_automation_repo}}
```

Comment:
was this incorrect before?

Reply:
@skazi0 yeah, what basically happened is when you convert a JJB job into a template, you have to double all your `{`s  and `}`s, otherwise jenkins-job-builder will think you're trying to expand a template parameter